components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -77,31 +146,55 @@ service Policies {
     option (chef.automate.api.policy).action = "update";
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}";
     option (chef.automate.api.iam.policy).action = "iam:policies:update";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  Get IAM version
+
+  Returns the major and minor version of IAM that your automate installation is running.
+  */
   rpc GetPolicyVersion (GetPolicyVersionReq) returns (GetPolicyVersionResp) {
     option (google.api.http) = {
       get: "/iam/v2/policy_version"
       additional_bindings {
-        get: "/iam/v2beta/policy_version"     
+        get: "/iam/v2beta/policy_version"
       }
     };
     option (chef.automate.api.policy).resource = "auth:policies";
     option (chef.automate.api.policy).action = "read";
     option (chef.automate.api.iam.policy).resource = "iam:policyVersion";
     option (chef.automate.api.iam.policy).action = "iam:policies:get";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  List policy members
+
+  List all members of a specific policy.
+  */
   rpc ListPolicyMembers (ListPolicyMembersReq) returns (ListPolicyMembersResp) {
     option (google.api.http) = {
       get: "/iam/v2/policies/{id}/members"
       additional_bindings {
-        get: "/iam/v2beta/policies/{id}/members"    
+        get: "/iam/v2beta/policies/{id}/members"
       }
     };
     option (chef.automate.api.policy).resource = "auth:policies:{id}";
     option (chef.automate.api.policy).action = "read";
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}:members";
     option (chef.automate.api.iam.policy).action = "iam:policyMembers:get";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  Replace policy members
+
+  Replace the entire member list for a specific policy with a new list.
```

Comment:
Add after this:
"Ensure each element of the `members` array is in the correct [Member Expression]({{< relref "iam-v2-guide.md#member-expressions" >}}) format."

Is that hugo link going to work here? I don't know. 🤷‍♂ 

That sentence should be added to Replace, Add, and Remove members.